src/services/mentor.service.ts

```diff
@@ -0,0 +1,75 @@
+import { dataSource } from '../configs/dbConfig'
+import Mentor from '../entities/mentor.entity'
+import Profile from '../entities/profile.entity'
+import { ApplicationStatus } from '../enums'
+import Category from '../entities/category.entity'
+
+export const createMentor = async (req: any) => {
```

Comment:
Pass only the user and the application here.
```suggestion
export const createMentor = async (user: Profile, application: MentorApplication) => {
```